hbw/selection/default.py

```diff
@@ -184,11 +185,11 @@ def default(
 
 
 @selector(
-    uses={default, gen_hbw_decay_products},
-    produces={category_ids, process_ids, increment_stats, gen_hbw_decay_products},
+    uses={default, gen_hbw_decay_products, gen_hbw_decay_features, "mc_weight"},
```

Comment:
I think we don't need `gen_hbw_decay_products`, `gen_hbw_decay_features` in `uses` here